### ISSUE TITLE
Build failure, FreeBSD 10.2

### ISSUE THREAD

**Issue by pdostal** [cocodelabs/znc-palaver#36]
Hello

I'd like to use Palaver, but I run ZNC on FreeBSD machine and I got this output from ```make```.

```
make: "/usr/local/etc/znc/znc-palaver/Makefile" line 4: Need an operator
make: "/usr/local/etc/znc/znc-palaver/Makefile" line 6: Need an operator
make: Fatal errors encountered -- cannot continue
make: stopped in /usr/local/etc/znc/znc-palaver
```

Can you please help me?

**Comment by kylef** [comment]
Sorry about this, I've fixed an issue with FreeBSD compatibility in 482bcf07804a7745a08cd799d997bdebc9e8bd36. Could you give this a try again please.

**Comment by pdostal** [comment]
That's awesome, I just went for a beer and you fixed it for me! :smile:
```
Building "palaver.so" for ZNC 1.6.1... [ ok ]
```
